src/js/player.js

```diff
@@ -3267,9 +3262,12 @@ class Player extends Component {
     }
 
     let techAutoplay;
+    // manual autoplay is only supported where promises are
+    const PromiseClass = this.options_.Promise || window.Promise;
```

Comment:
I would just call this `Promise`

Reply:
I just copy pasted what we used in `play` I think there may have been `nodejs` require issues calling it just `Promise`

Reply:
carry on then